src/commcareConnect.js

```diff
@@ -222,7 +300,9 @@ define([
         parseBindElement: function (form, el, path) {
             let mug = form.getMugByPath(path);
             if (!mug) {
+                // check each mugConfig to see if this is path matches
```

Comment:
Not sure of correct wording here. Is this suggestion correct?
```suggestion
                // check each mugConfig to see if this path matches
```